packages/global-nav/components/GlobalNav.js

```diff
@@ -235,7 +208,8 @@ GlobalNav.defaultProps = {
   onToggleResultClick: () => { },
   search: "",
   hamburgerMenu: true,
-  searchComponent: null
+  searchComponent: null,
+  notificationURL: "https://naas.gccollab.ca"
```

Comment:
anything we can do to not have to hard-code this?

Reply:
Right now it defaults to the production url but you can pass a different url to the component in the directory frontend.